tfimm/architectures/lora/convnext.py

```diff
@@ -13,37 +14,32 @@
 class LoRAConvNeXtConfig(ConvNeXtConfig):
     lora_rank: int = 4
     lora_alpha: float = 1.0
+    lora_train_bias: str = "none"
     # TODO: lora_dropout
 
 
 @keras_serializable
 @register_lora_architecture
 class LoRAConvNeXt(ConvNeXt):
     cfg_class = LoRAConvNeXtConfig
-    keys_to_ignore_on_load_missing = ["kernel_lora"]
 
     def __init__(self, cfg: LoRAConvNeXtConfig, **kwargs):
         # We first create the original model
         super().__init__(cfg, **kwargs)
+        self.cfg = cfg
```

Comment:
Yes, it is. But in order for PyCharm to know that `self.cfg` is of type `LoRAConvNeXtConfig` rather than `ConNeXtConfig` I added this line here. Otherwise I get red wavy lines when I access `self.cfg.lora_train_bias`. If there is another way, I'd be happy to change it.